src/Processor/Braintree/Mapper/PaymentMethodMapper.php

```diff
@@ -37,6 +49,17 @@ public function fromResult($paymentMethod): Token
         $isDefault = $paymentMethod->isDefault();
         $token->setIsDefaultPaymentMethod($isDefault);
 
+        $billingAddress = $paymentMethod->billingAddress;
+        if ($billingAddress instanceof BraintreeAddress) {
+            $state = null;
+            try {
+                $state = new State($billingAddress->region);
+            } catch (TypeError $e) {
+                // we tried
+            }
+            $token->setBillingAddress(new Address($state, $billingAddress->postalCode));
```

Comment:
Does this mean that the state coming back from BT will override anything we set previously? Do I need to set it previously?